tox_encryptsave/src/lib.rs

```diff
@@ -33,7 +33,7 @@ use tox_crypto::{
     NONCEBYTES, MACBYTES,
     Nonce, PrecomputedKey,
     gen_nonce,
-    sha256, memzero
```

Comment:
Do we use memzero somewhere else?

Reply:
No.

Reply:
@ unseddd how about removing memzero from tox_crypto if we dont use it?

Reply:
> @unseddd how about removing memzero from tox_crypto if we dont use it?

No problem, wanted to make minimal changes. Can remove `memzero` from the rest of the crate.

Will fix my commit message with the update :)